great_expectations/expectations/metrics/column_aggregate_metrics/column_sample_values.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+from great_expectations.compatibility.sqlalchemy import sqlalchemy as sa
+from great_expectations.core.metric_domain_types import MetricDomainTypes
+from great_expectations.execution_engine import (
+    SqlAlchemyExecutionEngine,
+)
+from great_expectations.expectations.metrics.column_aggregate_metric_provider import (
+    ColumnAggregateMetricProvider,
+)
+from great_expectations.expectations.metrics.metric_provider import metric_value
+
+
+class ColumnSampleValues(ColumnAggregateMetricProvider):
+    metric_name = "column.sample_values"
+
+    @metric_value(engine=SqlAlchemyExecutionEngine)
+    def _sqlalchemy(
+        cls,
+        execution_engine: SqlAlchemyExecutionEngine,
+        metric_domain_kwargs: dict,
+        metric_value_kwargs: dict,
+        **kwargs,
+    ):
+        selectable, _, _ = execution_engine.get_compute_domain(
+            domain_kwargs=metric_domain_kwargs, domain_type=MetricDomainTypes.COLUMN
+        )
+        column = metric_domain_kwargs["column"]
+        count = metric_value_kwargs["count"]
+        query: sa.Select = sa.select(sa.column(column)).select_from(selectable).limit(count)  # type: ignore[arg-type] # FIXME CoP
```

Comment:
You can find this same type: ignore all over the codebase. I tried to fix it, but `get_compute_domain` needs to be refactored first because it returns a union with lots of non-overlapping types.